docs/release-notes/3.0/release-3.0.0.md

```diff
@@ -0,0 +1,140 @@
+- **Update to latest version of TorchSharp** ([#6636](https://github.com/dotnet/machinelearning/pull/6636)) - Updated to the latest version of TorchSharp and fixed any breaking changes so we can take advantage of their new features and bug fixes.
+- **Update to latest version of Onnx Runtime** ([#6624](https://github.com/dotnet/machinelearning/pull/6624)) - Updated to the latest version of Onnx Runtime and fixed any breaking changes so we can take advantage of their new features and bug fixes.
+- **Update ML.NET to compile with .NET8** ([#6641](https://github.com/dotnet/machinelearning/pull/6641)) - Removed some deprecated code now throws errors on .NET8 as well as other minor fixes to allow working/building with .NET8.
+- **Added more logging to Object Detection** ([#6646](https://github.com/dotnet/machinelearning/pull/6646)) - Added more logging while Object Detection is training so even if epochs take a long time you can be sure things are still moving.
+- **Update timeout error message in AutoMLExperiment** ([#6613](https://github.com/dotnet/machinelearning/pull/6613)) - Updated the error message so it is more clear what happened.
+- **Add batchsize and arch to imageClassification SweepableTrainer** ([#6597](https://github.com/dotnet/machinelearning/pull/6597)) - Added batchsize and arch to the ImageClassification SweepableTrainer so those can now be trained on.
+- **Update max_model when trial fails** ([#6596](https://github.com/dotnet/machinelearning/pull/6596))
+- **Add default search space for standard trainers** ([#6576](https://github.com/dotnet/machinelearning/pull/6576)) - Added a default search space for all standard trainers so users have reasonable default values.
+- **Adding more metrics to BinaryClassification Experiment** ([#6571](https://github.com/dotnet/machinelearning/pull/6571))
+- **Add checkAlive in NasBertTrainer** ([#6546](https://github.com/dotnet/machinelearning/pull/6546)) - Now we check between batches if cancellation was requested and stop processing if so.
+- **OneDAL - Fallback to default implementation** ([#6538](https://github.com/dotnet/machinelearning/pull/6538)) - If you specify you want to use OneDal but something happens that prevents you from using it, like it can't find the binaries/etc, it will auto default back to the normal implementation instead of crashing.
+- **Add addKeyValueAnnotationsAsText flag in AutoML** ([#6535](https://github.com/dotnet/machinelearning/pull/6535))
+- **Add continuous resource monitoring to AutoML.IMonitor** ([#6520](https://github.com/dotnet/machinelearning/pull/6520)) - Thanks @andrasfuchs!
+- **Update WebClient to HttpClient implementations** ([#6476](https://github.com/dotnet/machinelearning/pull/6476)) - Update a usage of WebClient to HttpClient since WebClient is now deprecated. Thanks @rgesteve!
```

Comment:
>a [](http://example.com/codeflow?start=125&length=1)

the

Reply:
```suggestion
- **Update WebClient to HttpClient implementations** ([#6476](https://github.com/dotnet/machinelearning/pull/6476)) - Update usage of WebClient to HttpClient since WebClient is now deprecated. Thanks @rgesteve!
```